blooming_history/blooming_history.c

```diff
@@ -157,13 +157,16 @@ void *pthread_entry_upload(void *attr)
    struct bloom *bloom_send;
    CURL *curl = NULL;
    int error = 0;
+   char url[320];
```

Comment:
This is not a good practice... You should define a constant with some self-explaining name and use it.

Also, you should replace `sprintf()` with `snprintf()` (with the size constant) on line: https://github.com/CESNET/Nemea-Modules/blob/3f759a529d775ef1c232bc672c3ed16e6570c02e/blooming_history/blooming_history.c#L187

Reply:
I see `AGGREGATOR_SERVICE` can be passed by user! You really must use safe `snprintf()`.